performance_tests/conftest.py

```
@@ -0,0 +1,107 @@
+    """Set up comparison logging libraries."""
+    loggers = {}
+    
+    try:
+        # Standard library logging
+        import logging
+        logging.basicConfig(level=logging.INFO)
+        std_logger = logging.getLogger("stdlib_test")
+        std_logger.addHandler(logging.NullHandler())
+        std_logger.propagate = False
+        loggers["standard_library"] = std_logger
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)